--- x/qtbase/mkspecs/common/g++-win32.conf
+++ y/qtbase/mkspecs/common/g++-win32.conf
@@ -61,7 +61,7 @@
 QMAKE_LIB_EXTENSIONS    = a dll.a

 QMAKE_LIBS              =
-QMAKE_LIBS_GUI          = -lgdi32 -lcomdlg32 -loleaut32 -limm32 -lwinmm -lws2_32 -lole32 -luuid -luser32 -ladvapi32
+QMAKE_LIBS_GUI          = -lgdi32 -lcomdlg32 -loleaut32 -limm32 -lwinmm -lws2_32 -lole32 -luuid -luser32 -ladvapi32 -liphlpapi -luserenv -lpsapi -lwtsapi32 -lnetapi32
 QMAKE_LIBS_NETWORK      = -lws2_32
 QMAKE_LIBS_OPENGL       = -lglu32 -lopengl32 -lgdi32 -luser32
 QMAKE_LIBS_OPENGL_ES2   = -lgdi32 -luser32